src/Data/Lens/At.purs

```diff
@@ -0,0 +1,45 @@
+module Data.Lens.At
+  ( At
+  , at
+  ) where
+
+import Prelude
+
+import Data.Identity (runIdentity, Identity(..))
+import Data.Map as M
+import Data.Maybe (Maybe(..), maybe)
+import Data.Set as S
+import Data.StrMap as SM
+import Data.Tuple
+
+import Data.Lens (LensP(), lens)
+import Data.Lens.Index (Index)
+
+class (Index m a b) <= At m a b where
```

Comment:
Yep, for `String` and `Char`, for example, it makes sense to keep it this way.

Sounds good, but do you see any benefit to `IndexKey` etc.?